pootle/templates/editor/units/xhr_tm.html

```diff
@@ -10,8 +10,14 @@
         <div class="suggestion-translation" lang="<%= store.target_lang %>"
           dir="<%= store.target_dir %>"><%= PTL.utils.fancyHl(sugg.target) %></div>
         <% if (sugg.project) { %>
+        <% if (sugg.submitted_on) { %>
+        <div class="tm-context">
+          <%= interpolate(gettext('Translated by %(fullname)s in &ldquo;<span title="%(path)s">%(project)s</span>&rdquo; project <time class="extra-item-meta js-relative-date" title="%(submitted_format)s" datetime="%(submitted_isoformat)s">&nbsp;</time>'), {fullname: sugg.fullname, path: sugg.path, project: sugg.project, submitted_format: sugg.submitted_on, submitted_isoformat: sugg.submitted_on|isoformat}, true) %>
```

Comment:
Can we abstract the HTML tags away so we reduce the risk of localizers breaking actual functionality? Note the whole `<time ....>&nbsp;</time>` is avoidable noise we can reduce to a placeholder.